apps/src/code-studio/pd/application_dashboard/cohort_view_table.jsx

```diff
@@ -0,0 +1,62 @@
+import React, {PropTypes} from 'react';
+import {Table} from 'reactabular';
+
+export default class CohortViewTable extends React.Component {
+  static propTypes = {
+    data: PropTypes.array.isRequired,
+  }
+
+  constructColumns() {
+    return [
+      {
+        property: 'date_accepted',
+        header: {
+          label: 'Date Accepted'
+        },
+        cell: {
+          format: (date_accepted) => {
+            return new Date(date_accepted).toLocaleDateString('en-us', {month: 'long', day: 'numeric'});
+          }
+        }
+      }, {
+        property: 'applicant_name',
+        header: {
+          label: 'Name'
+        }
+      }, {
+        property: 'district_name',
+        header: {
+          label: 'School District'
+        }
+      }, {
+        property: 'school_name',
+        header: {
+          label: 'School Name'
+        }
+      }, {
+        property: 'email',
+        header: {
+          label: 'Email'
+        }
+      }, {
+        property: 'registered_for_summer_workshop',
+        header: {
+          label: 'Registered Summer Workshop'
+        }
+      }
+    ];
+  }
+
+  render() {
+    return (
+      <Table.Provider
+        className="pure-table table-striped"
+        columns={this.constructColumns()}
+        style={{width: '100%'}}
```

Comment:
Our style guide prefers style objects defined outside of the render method for static styles: https://github.com/code-dot-org/code-dot-org/blob/staging/STYLEGUIDE.md#react